.circleci/config.yml

```diff
@@ -1,23 +1,47 @@
 version: 2.1
 
-orbs:
-  python: circleci/python@1.4.0
+jobs:
+  build_and_test:
```

Comment:
We do use docker, it just gets defined lower down than you might see elsewhere because I have defined the `parameters` at the top. This lists the values you can define underneath a job https://circleci.com/docs/2.0/configuration-reference/#jobs